docs/debezium-aws-rds-config-scripts

```diff
@@ -0,0 +1,361 @@
+	-#host    replication     postgres        127.0.0.1/32            trust
+	-#host    replication     postgres        ::1/128                 trust
+	host     replication     postgresuser    0.0.0.0/0               md5
+	*pg_hba.conf vi editor*
+    
+
+15. open ***postgresql.conf*** config file and change it as follows
+
+----
+    -bash-4.2$ vim postgresql.conf
+    ------ postgresql.conf vi editor ------
+	listen_addresses = '*'
+	port = 5432
+	ssl = off
+	wal_level = logical
+	max_wal_senders = 1
+	max_replication_slots = 1
+	------ postgresql.conf vi editor ------
+----
+
+
+16. Restart the server to apply changes
+
+    `-bash-4.2$ /usr/lib64/pgsql96/bin/pg_ctl restart -D /var/lib/pgsql96/data`
+
+17. Connect to the new db with the new user
+
+    `-bash-4.2$ psql -d inventory -U postgresuser`
+
+18. Give ***REPLICATION*** and ***SUPERUSER***(not mandatory) rights to the new user.
+
+----
+    testdb=> ALTER ROLE postgresuser WITH SUPERUSER;
+    testdb=> ALTER ROLE postgresuser WITH REPLICATION;
+----
+
+19. Add your user postgres to the ***sudoers*** group
+
+----
+    [ec2-user@hostname ~] $ usermod -aG wheel postgres
+    [ec2-user@hostname ~] $ cat /etc/sudoers
+----
+
+20. Allow wheel user group to run all commands on linux via ***visudo*** command
+
+----
+    [ec2-user@hostname ~] $ visudo`
+    ------* visudo editor for sudo ------***`
+	 -# Open wheel line`
+	 -## Allows people in group wheel to run all commands`
+	 %wheel        ALL=(ALL)       ALL`
+	------* visudo ------***
+----
+
+21. Find postgres development tools to ***compile wal2json plugin***
+
+----
+    [ec2-user@hostname ~] $ yum list postgres*
+    [ec2-user@hostname ~] $ sudo yum install postgresql96-devel.x86_64
+    [ec2-user@hostname ~] $ yum list gcc*
+    [ec2-user@hostname ~] $ sudo yum install gcc.noarch
+----
+
+22. Install git tool to ***download wal2json plugin code*** from the repository below.
+
+----
+    [ec2-user@hostname ~] $ sudo yum install git
+    [ec2-user@hostname ~] $ sudo -i -u postgres
+    -bash-4.2$ git clone https://github.com/eulerto/wal2json.git
+----
+
+23. ***Compile wal2json plugin***
+
+----
+    -bash-4.2$ cd wal2json/
+    -bash-4.2$ PATH=/path/to/bin/pg_config:$PATH
+    -bash-4.2$ USE_PGXS=1 make
+    -bash-4.2$ sudo /usr/bin/install -c -m 755  wal2json.so '/usr/lib64/pgsql96/'
+    > The same thing with (USE_PGXS=1 make install )
+----
+
+
+24. ***Create a replication slot*** named ***wal2json_rds*** using the plugin ***wal2json***.
+
+----
+    -bash-4.2$ psql inventory -U postgresuser
+    testdb=> SELECT * FROM pg_create_logical_replication_slot('wal2json_rds', 'wal2json');
+    testdb=> SELECT * FROM pg_replication_slots;
+----
+
+25. Check out your ***wal_level***, you must see ***logical***
+
+    `testdb=> show wal_level;`
+
+---
+## RESULT
+
+** As you can see, the records inserted from AWS RDS PostgreSQL DB are replicated to the target PostgreSQL DB.**
```

Comment:
This seems to be a bit disconnected to the steps before which describe how to setup a non-RDS Postgres as a sink.

Reply:
Yes, I realized it, but the article became too long, as you said. What do you recommend?